static/app.css

```diff
@@ -398,3 +398,13 @@ footer {
         margin-right: 20px;
     }
 }
+
+
+.centermode li{
+    /* width: 220px;
```

Comment:
@Saturate this is fixed. I have refactored the code and made minimal changes to the original lory.js code. This will be easy for you to review. Let me know if you need an explanation for any code update. I haven't added comments to keep it clean.